src/scripts/components/main/data-viewer/data-viewer.tsx

```diff
@@ -23,16 +24,23 @@ import {globeSpinningSelector} from '../../../selectors/globe/spinning';
 
 import {GlobeView} from '../../../types/globe-view';
 import {Marker} from '../../../types/marker-type';
+import {LayerType} from '../../../types/globe-layer-type';
 
-import styles from './globes.styl';
+import styles from './data-viewer.styl';
```

Comment:
Can you please move the styles import to the end. We tried to have the same import structure everywhere. Which is:
* node modules/native imports
* own modules
* types
* style imports

each block separated with a new line